tensorflow/contrib/learn/python/learn/datasets/base.py

```diff
@@ -144,3 +144,34 @@ def maybe_download(filename, work_directory, source_url):
         size = f.size()
       print('Successfully downloaded', filename, size, 'bytes.')
   return filepath
+
+def split_into_train_and_test(dataset, test_ratio=0.3, shuffle=False):
```

Comment:
I am not sure I understand the comment -- should I pull the splitting routine from the `_sklearn` completely and place it here, or should I just refer to `_sklearn` from here.